src/Umbraco.Web/Editors/EntityController.cs

```diff
@@ -43,6 +45,20 @@ namespace Umbraco.Web.Editors
     [PluginController("UmbracoApi")]
     public class EntityController : UmbracoAuthorizedJsonController
     {
+
+        /// <summary>
+        /// Configures this controller with a custom action selector
+        /// </summary>
+        private class EntityControllerConfigurationAttribute : Attribute, IControllerConfiguration
+        {
+            public void Initialize(HttpControllerSettings controllerSettings, HttpControllerDescriptor controllerDescriptor)
+            {
+                controllerSettings.Services.Replace(typeof(IHttpActionSelector), new ParameterSwapControllerActionSelector(

```

Comment:
ModelBinder is different from Action Selection. ModelBinder binds attributes - which I've had to create them for to support the GET/POST scenario already. Without a custom action selector you'll get: ambiguous actions detected. This is a custom selector that knows how to negotiate which actions to choose based on the values in the request.